pkg/telemetry/metrics_helper.go

```diff
@@ -61,7 +61,7 @@ func computeEndpointSHAForTanzuContext(ctx *configtypes.Context) string {
 	// returns SHA256 of concatenated string of Endpoint and orgId/ProjectName/SpaceName
 	return hashString(ctx.GlobalOpts.Endpoint +
 		stringValue(ctx.AdditionalMetadata[configlib.OrgIDKey]) +
-		stringValue(ctx.AdditionalMetadata[configlib.ProjectNameKey]) +
+		stringValue(ctx.AdditionalMetadata[configlib.ProjectIDKey]) +
```

Comment:
With backwards-compatibility support now, the `ctx.AdditionalMetadata[configlib.ProjectIDKey]` could be empty, right?  So we should pick the one that is filled between the project name or id I think

Reply:
Yes, I missed this after the backward compatibility changes. I will update it. Thanks